packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/document/DocumentExtraction.java

```diff
@@ -12,10 +12,16 @@
 @Accessors(chain = true)
 public class DocumentExtraction implements Serializable {
 
-    @JsonAlias("file_name")
+	public static final String TABLE_ASSETTYPE = "table";
+	public static final String EQUATION_ASSETTYPE = "equation";
+	public static final String FIGURE_ASSETTYPE = "figure";
```

Comment:
Why are they lowercase here and Capital case in front-end?

Reply:
Thanks! back end isn't caring about case, but, they should be the same. fixed.